### PR TITLE
Automatically load override file if it exists

### LINKED ISSUE #16: Read override files.
Would be nice if the parsing of the yml took into account the override.yml as well.

### PR DESCRIPTION
**Pull request by jubianchi**
...or ignore it using `--ignore-override`

Closes #16

### CODE REVIEW

File: src/application.php [hunk 1]
```diff
@@ -45,6 +49,13 @@
         }
 
         $configuration = readConfiguration($inputFile);
+
+        if (!$input->getOption('ignore-override') && file_exists($overrideFile)) {
+            $override = readConfiguration($overrideFile);
+
+            $configuration = array_merge_recursive($configuration, $override);
```
Comment (osterrfr-crim) [inline src/application.php]: This might be the culprit for not merging both configurations...

Comment (jubianchi) [inline src/application.php]: The question here is should we use `array_merge_recursive` or `array_replace_recursive` ?

Comment (artospaj) [inline src/application.php]: @jubianchi 
I guess it's not that simple. Some properties should be overwritten (i.e. image definition), some should be merged (i.e. list of links), others - replaced..

I'm working on merging multiple docker-compose.yml files into the big one - that can be used here as input. 
So far it's developed internally, but I'll let you know as soon as it's published on github.

Comment (jubianchi) [inline src/application.php]: @astrospaj I implemented a merge strategy that seems to work pretty well. Everything is merge except form volumes and links where I coded a conflict resolution.

If two volume map to the same container directory, only the last one will be present. The same applies for link aliases.

**Pull request merged**